terraform/modules/common/ssh_key/main.tf

```diff
@@ -0,0 +1,18 @@
+## Variables
+
+variable "stack_name" {
+  description = "The name of the stack within which the resource resides"
+}
+
+## Resources
+
+resource "aws_key_pair" "ssh_key" {
```

Comment:
Is there a particular advantage to having the ssh_key as it's own file? It feels like it's closely enough tied to the jumpbox (as in we have no other need for one) that it could go into that file in the same way the security groups do

Reply:
Could be in the jumpbox project